operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md

```diff
@@ -0,0 +1,198 @@
+# Update default ServerTech PDU credentials used by the Redfish Translation Service
+
+This procedure updates the default credentials used by the Redfish Translation Service (RTS) for when new ServerTech PDUs are discovered in a system.
+
+The Redfish Translation Service provides a Redfish interface that CSM Services (HSM, and CAPMC) can use interact with ServerTech PDUs which do not natively support Redfish. 
```

Comment:
- Change "The Redfish Translation Service" to RTS
- Change "that CSM Services (HSM, and CAPMC)" to "that the Hardware State Manager (HSM) and Cray Advanced Platform Monitoring and Control (CAPMC) services"